src/mixins/document-mixin.ts

```diff
@@ -37,12 +42,17 @@ export default class DocumentMixin extends Vue {
           throw new Error('Invalid API response')
         }
         return data
-      }).catch(error => {
-        console.log('Error getting presigned url =', error) // eslint-disable-line no-console
-        throw error
```

Comment:
This is redundant. The Axios error was already thrown and displayed a console message..